tests/certification/bindings/azure/storagequeues/storagequeue_test.go

```diff
@@ -0,0 +1,409 @@
+		msgsFor2 := make([]string, numOfMessages/2)
+		for i := 0; i < numOfMessages/2; i++ {
+			msgsFor1[i] = fmt.Sprintf("standard-binding-1: Message %03d", i)
+		}
+
+		for i := numOfMessages / 2; i < numOfMessages; i++ {
+			msgsFor2[i-(numOfMessages/2)] = fmt.Sprintf("standard-binding-2: Message %03d", i)
+		}
+
+		messagesFor1.ExpectStrings(msgsFor1...)
+		messagesFor2.ExpectStrings(msgsFor2...)
+
+		metadata := make(map[string]string)
+
+		ctx.Log("Invoking binding 1!")
+		for _, msg := range msgsFor1 {
+			ctx.Logf("Sending: %q", msg)
+
+			req := &daprClient.InvokeBindingRequest{Name: "standard-binding-1", Operation: "create", Data: []byte(msg), Metadata: metadata}
+			err := client.InvokeOutputBinding(ctx, req)
+			require.NoError(ctx, err, "error publishing message")
+		}
+
+		ctx.Log("Invoking binding 2!")
+		for _, msg := range msgsFor2 {
+			ctx.Logf("Sending: %q", msg)
+
+			req := &daprClient.InvokeBindingRequest{Name: "standard-binding-2", Operation: "create", Data: []byte(msg), Metadata: metadata}
+			err := client.InvokeOutputBinding(ctx, req)
+			require.NoError(ctx, err, "error publishing message")
+		}
+
+		// Assertion on the data.
+		messagesFor1.Assert(ctx, time.Minute)
+		messagesFor2.Assert(ctx, time.Minute)
+
+		return nil
+	}
+
+	application := func(ctx flow.Context, s common.Service) (err error) {
+		// Setup the input binding endpoints.
+		err = multierr.Combine(err,
+			s.AddBindingInvocationHandler("standard-binding-1", func(_ context.Context, in *common.BindingEvent) ([]byte, error) {
+				messagesFor1.Observe(string(in.Data))
+				ctx.Logf("Got message: %s", string(in.Data))
+				return []byte("{}"), nil
+			}),
+			s.AddBindingInvocationHandler("standard-binding-2", func(_ context.Context, in *common.BindingEvent) ([]byte, error) {
+				messagesFor2.Observe(string(in.Data))
+				ctx.Logf("Got message: %s", string(in.Data))
+				return []byte("{}"), nil
+			}))
+		return err
+	}
+
+	flow.New(t, "servicebusqueue certification").
+		// Run the application logic above.
+		Step(app.Run("standardApp", fmt.Sprintf(":%d", appPort), application)).
+		Step(sidecar.Run("standardSidecar",
+			embedded.WithAppProtocol(runtime.HTTPProtocol, appPort),
+			embedded.WithDaprGRPCPort(grpcPort),
+			embedded.WithDaprHTTPPort(httpPort),
+			embedded.WithComponentsPath("./components/standard"),
+			runtime.WithOutputBindings(
+				binding_loader.NewOutput("azure.storagequeues", func() bindings.OutputBinding {
+					return binding_asq.NewAzureStorageQueues(log)
+				}),
+			),
+			runtime.WithInputBindings(
+				binding_loader.NewInput("azure.storagequeues", func() bindings.InputBinding {
+					return binding_asq.NewAzureStorageQueues(log)
+				}),
+			),
+			runtime.WithSecretStores(
+				secretstores_loader.New("local.env", func() secretstores.SecretStore {
+					return secretstore_env.NewEnvSecretStore(log)
+				}),
+			))).
+		Step("interrupt network", network.InterruptNetwork(time.Minute, []string{}, []string{}, "5671", "5672")).
```

Comment:
Make this port 443 - since that is what Azure Storage Queues uses (REST).

Also, can you send some data (use the output binding first), then interrupt, then complete your test please? Just to show that the component worked prior to the interruption also.